dc/core/FFMPEGVideoStream.cpp

```diff
@@ -242,6 +242,11 @@ void FFMPEGVideoStream::_openVideoStreamDecoder()
     if( !codec )
         throw std::runtime_error( "No decoder found for video stream" );
 
+    // multi-threaded decoding is only useful for some h264 movies, according
+    // to common believe on the internet. 1 could mean 0/auto, hence all CPU threads,
```

Comment:
to common believe -> to a common belief
I am not sure I get the second part: "1 could mean 0/auto";  1 IS the same as 0==auto so we can't use it? Or we just use 2 threads to have a bit more "power" for big movies?